.github/workflows/audit-bootnodes.yml

```diff
@@ -0,0 +1,77 @@
+name: Audit Bootnodes
+
+on:
+
+  schedule:
+    - cron: '0 6 * * *'
+
+  pull_request:
+    branches:
+      - 'master'
+    paths:
+      - 'params/bootnode*'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+
+  build:
+    name: Audit Bootnodes
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.15
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Build bin/devp2p
+        run: |
+          mkdir -p build/bin
+          go build -o build/bin/devp2p ./cmd/devp2p
+
+      - name: Audit Bootnodes
+        shell: bash
+        run: |
+          healthcheck() {
+            enode_list="$(cat "$1" |grep -E 'enode://' | sed 's-"--g' | sed 's-\t--g'| sed 's/,.*//g')"
+            echo "$enode_list" | while read -r line; do
+              if >/dev/null 2>&1 ./build/bin/devp2p discv4 ping "${line}"; then
+                >&2 echo "PASS: ${line}"
```

Comment:
Woow, nice use of redirection. I am not good at bash and I am learning new stuff by checking your code @meowsbits  :)